### PR TITLE
Feature/updatebf

### PR DESCRIPTION
**Pull request by mc2922**
updated for deprecation warnings 
ready for 1.0?

### CODE REVIEW

File: test/beamforming/testFwdBackConv.jl [hunk 1]
```diff
@@ -1,77 +1,41 @@
...
-# setVal!(fg, :x7, zeros(2,N))
-# addVariable!(fg, :x8, Point2)
-# setVal!(fg, :x8, zeros(2,N))
-# addVariable!(fg, :x9, Point2)
-# setVal!(fg, :x9, zeros(2,N))
-
-
-addVariable!(fg, :l1, Point2)
+#landmark
+addNode!(fg, :l1, Point2)
```
Comment (dehann) [inline test/beamforming/testFwdBackConv.jl]: Hi @mc2922 , could we change these to all be `addVariable!(fg, :l1, Point2)` instead?

File: test/testdata/test_array_positions.csv [hunk 2]
```diff
@@ -0,0 +1,10 @@
+0.000000000000000000e+00,0.000000000000000000e+00,0.000000000000000000e+00
```
Comment (dehann) [inline test/testdata/test_array_positions.csv]: check with @nicrip on whether it is okay to publish this file?

**Comment by dehann** [comment]
Also, we are targeting IIF v0.7.2 now as well as RoME v0.4.0 tomorrow.  Caesar by Tuesday.

File: test/testdata/test_array_positions.csv (hunk 2 above)
Comment (mc2922) [reply test/testdata/test_array_positions.csv]: Oops sorry I thought the test files were in the repo already. Let's leave them out for now until I ask him.

File: test/beamforming/testFwdBackConv.jl (hunk 1 above)
Comment (mc2922) [reply test/beamforming/testFwdBackConv.jl]: changed landmark and locations in loop

**Pull request merged**